### PR TITLE
Issue #1780 - VowelsConsonants #DWOC

### PR DESCRIPTION
**Pull request by shloka-gupta**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

I added the Vowels and Consonants program

### Checklist

- [YES] I've read the contribution guidelines.
- [YES] I've checked the issue list before deciding what to submit.
- [YES] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #1780

### CODE REVIEW

**Comment by shloka-gupta** [comment]
Kindly tell me if there are any changes required

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to PEP8 guidelines. You can use [this](http://pep8online.com/).

**Comment by shloka-gupta** [comment]
I did so

**Comment by shloka-gupta** [comment]
Can someone kindly tell me there are any required changes?

File: Python/README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 - [Search Insert Position](cp/search_insert_position.py)
 - [Stock span problem](cp/stock_span.py)
 - [Bulb Switcher problem](cp/bulb_switcher.py)
+- [Vowels ans consonants in a word](cp/VowelsConsonants.py)
```
Comment (atarax665) [inline Python/README.md]:
```suggestion
- [Vowels and consonants in a string](cp/VowelsConsonants.py)
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/cp/VowelsConsants.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+"""
+The program helps to count the number of
+vowels and consants in a string.
+
+"""
+
+def Count(word):
```
Comment (HarshCasper) [inline Python/cp/VowelsConsants.py:8]: Add proper Docstrings

File: Python/cp/VowelsConsants.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                vc = vc + 1
+            else:
+                cc = cc + 1
+    return vc, cc
+
+TheWord = input("Enter a string: ").lower()
+
+vc, cc = Count(TheWord)
+
+print("Vowel count = ", vc, " Consonant count = ", cc)
```
Comment (HarshCasper) [inline Python/cp/VowelsConsants.py]: Enclose in a `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by shloka-gupta** [comment]
Re-review

**Review by atarax665 — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Comment by chicken-biryani** [comment]
Hey, can you add the DWOC label?

**Comment by shloka-gupta** [comment]
Hey @HarshCasper // @atarax665  can you kindly add the DWOC label?

**Pull request merged**